doc/man3/OSSL_SELF_TEST_set_callback.pod

```diff
@@ -0,0 +1,46 @@
+=pod
+
+=head1 NAME
+
+OSSL_SELF_TEST_set_callback,
+OSSL_SELF_TEST_get_callback - specify a callback for processing self tests
+
+=head1 SYNOPSIS
+
+ #include <openssl/self_test.h>
+
+ void OSSL_SELF_TEST_set_callback(OPENSSL_CTX *ctx, OSSL_CALLBACK *cb);
+ OSSL_CALLBACK *OSSL_SELF_TEST_get_callback(void);
+
+=head1 DESCRIPTION
+
+OSSL_SELF_TEST_set_callback() sets the function B<cb> that is called during
+self testing that is associated with a given library context B<ctx>.
```

Comment:
Link to the OPENSSL_CTX page?

Reply:
Added a link to the see also.. Seems redundant to have to add a link elsewhere in a pod with half a page of info in it.